forms.py

```diff
@@ -0,0 +1,37 @@
+from flask_wtf import FlaskForm
+from wtforms import SubmitField, BooleanField, StringField, PasswordField
+from wtforms.validators import DataRequired, ValidationError, EqualTo
+from wtforms_sqlalchemy.fields import QuerySelectMultipleField
+import app
+
+def actor_query():

```

Comment:
Šį kartą nieko nemažinsiu bet baigiamajam atkreip demėsį ka dėdį į kodą šitų fukcijos yra kodė tačiau jos nenaudojimos vadinamas kodo perteklius